cfgov/v1/wagtail_hooks.py

```diff
@@ -76,7 +76,7 @@ def editor_js():
 @hooks.register('insert_editor_css')
 def editor_css():
     css_files = [
-        'css/table-block.css',
+        'css/table-block.css', 'css/bureau-structure.css'
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/2532/commits/861674d42470d1ebe15c3ac204e66b4fa58f741c